### DISCUSSION TITLE
Source Folder?

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [tavianator/bfs#87] [Q&A] [answered]
Why is there no `/Source/` folder?

Having all source code in the root folder isn't exactly great..

**Comment by tavianator** [comment, marked as answer]
Good question!  `bfs` [started](https://github.com/tavianator/bfs/tree/72552f880f3ca52c0d98d875b1da783e5a2fa2e7) as a pretty small project and grew organically from there.  The root directory has grown pretty large since then, and it might be time for a `src/` folder.

On the other hand, C utilities dumping all their source files in the project root is pretty common.  The first example that comes to mind is [git](https://github.com/git/git/) itself.

**Reply by ElectronicsArchiver** [reply]
Great to hear it's not intentional,
in that case I'd fix that up in a PR?

**Reply by tavianator** [reply]
Sure, go for it!
